qiita_pet/templates/study_description_templates/raw_data_editor_tab.html

```diff
@@ -12,10 +12,12 @@
         <ol>
           <!-- Step 1: choose the prep template file -->
           <li>
-            select your prep template file
+            select your prep template file (only files with the "txt" and "tsv" file extensions will be displayed here):
             <select id="add_prep_template_{{raw_data_id}}">
               {% for f in files %}
+                {% if f.endswith(('txt', 'tsv')) %}
```

Comment:
I *think* I actually learned this from reviewing some of your code at some point last year, although I may be misremembering :laughing: